qgreenland/util/qgis/metadata.py

```diff
@@ -72,7 +72,7 @@ def _build_layer_tooltip(layer_cfg: Layer) -> str:
 
 
 def build_layer_abstract(layer_cfg: Layer) -> str:
```

Comment:
Perhaps we should also rename this function to `build_layer_metadata`? The string returned from this includes the layer description, dataset abstract, and citation information.

Reply:
:+1: The name of this function always bothered me.

Reply:
I think it was called `build_layer_abstract` originally because we use this text to populate the abstract section of the layer metadata in QGIS. Changed to `build_layer_metadata` in 732d657d.